lib/ecto/datetime.ex

```diff
@@ -214,12 +215,17 @@ defmodule Ecto.Time do
   end
 
   defp erl_load({_, {hour, min, sec}}) do
-    %Ecto.Time{hour: hour, min: min, sec: sec}
+    %Ecto.Time{hour: hour, min: min, sec: sec, usec: 0}
   end
 
   defimpl String.Chars do
-    def to_string(%Ecto.Time{hour: hour, min: min, sec: sec}) do
-      zero_pad(hour, 2) <> ":" <> zero_pad(min, 2) <> ":" <> zero_pad(sec, 2)
+    def to_string(%Ecto.Time{hour: hour, min: min, sec: sec, usec: usec}) do
+      str = zero_pad(hour, 2) <> ":" <> zero_pad(min, 2) <> ":" <> zero_pad(sec, 2)
+      if is_nil(usec) or usec == 0 do
```

Comment:
This is meant to be more of a human readable representation. I think we should leave usec out. Wdyt?